Dockerfile

```diff
@@ -15,18 +15,24 @@ ENV INITIAL_MEMORY="512m"
 ENV MAXIMUM_MEMORY="512m"
 ARG OVR=""
 ENV JAVA_OPTS="${JAVA_OPTS} -Xms${INITIAL_MEMORY} -Xmx${MAXIMUM_MEMORY}"
-ENV GEOSTORE_OVR_OPT="-Dgeostore-ovr=file://${CATALINA_BASE}/conf/${OVR}"
-ENV JAVA_OPTS="${JAVA_OPTS} ${GEOSTORE_OVR_OPT}"
+# ENV GEOSTORE_OVR_OPT="-Dgeostore-ovr=file://${CATALINA_BASE}/conf/${OVR}"
+ENV GEOSTORE_OVR_OPT=""
+ENV JAVA_OPTS="${JAVA_OPTS} ${GEOSTORE_OVR_OPT} -Ddatadir.location=${DATA_DIR}"
 ENV TERM xterm
 
+#name of the external dir
+ENV DATA_DIR="/data"
+RUN mkdir -p ${DATA_DIR}/configs
+#add all files you want to copy to the external dir
+COPY newext.json /data/configs/new.json 
 COPY --from=mother "/mapstore/mapstore.war" "${MAPSTORE_WEBAPP_DST}/mapstore.war"
 COPY --from=mother "/mapstore/docker" "${CATALINA_BASE}/docker/"
 
 WORKDIR ${CATALINA_BASE}
 
 RUN cp ${CATALINA_BASE}/docker/wait-for-postgres.sh /usr/bin/wait-for-postgres
 
-RUN cp ${CATALINA_BASE}/docker/${OVR} ${CATALINA_BASE}/conf
+# RUN cp ${CATALINA_BASE}/docker/${OVR} ${CATALINA_BASE}/conf
```

Comment:
if we are using the datadir for the datadir now, why not copy the file in there?